scripts/dev-utils/batchUuidUpdate.js

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env node
+
+'use strict';
+
+const {purgeAmp} = require('../../worker/updates');
+const contentToBePurged = require('./uuid-set').uuids;
```

Comment:
instead of this being a hardcoded list, can it take a command line argument that's e.g. a path to a text file to read the list from?

Reply:
Then wouldn't I start needing to include file buffer parsers libraries etc etc?

Reply:
nah, node has enough built in

```js
const fs = require('fs')
const uudis = fs.readFileSync(process.argv[1], 'utf8').split('\n')
```